endpoints.md

```diff
@@ -12,4 +12,42 @@
 * /api/core/sites
 
 ## Proposed Endpoints
-* /api/discover
+* /api/discover/(dso)?(sort)
```

Comment:
/api/discover/search
- to provide access the the discovery search. The supported parameters could be:
* profile (similar to the current location parameter but more general so that we can provide different configuration for the repository, for a community, etc.)
* facet (values to apply, to exclude, etc.)
* query (the generic query string to apply)
* sort (the sort conditions)

/api/discover/browse
- to provide access to the browse system. it should return the list of available browse indexes. On a specific browse indexes /api/discover/browse/<index-name> the supported parameters could be:
* sort
* filter value (on metadata browse)